release/models/platform/openconfig-platform-integrated-circuit.yang

```diff
@@ -102,16 +108,16 @@ module openconfig-platform-integrated-circuit {
       units "bits per second";
       description
         "Total backplane-facing capacity that is currently available based
-        on the active links.";
+        on the active fabric links.";
       oc-ext:telemetry-on-change;
     }
 
     leaf consumed-capacity {
       type uint64;
       units "bits per second";
       description
-        "Backplane-facing capacity that is consumed by front-panel ports that are connected
-        to the integrated circuit and are operationally up.";
+        "Backplane-facing capacity that is consumed by the traffic on front-panel ports
```

Comment:
It's not strictly the traffic only those ports, right? If I have 400G front-panel port this consumes 400G, not whatever the traffic load on that port is at this exact moment in time.

Reply:
Rohit and I discussed an issue brought up by implementors of this model.  The purpose of this PR is to explore and  clarify what the intent of this leaf is.   The modern, modular form factor packet based devices that I am familiar with only consume fabric capacity based on packets or cells sent over the fabric.  Bringing a front panel port up or down does not change the capacity that is in consumed between the INTEGRATED_CIRCUIT (IC) and the FABRIC.  

@rohit-rp I added a Juniper reference to the PR description.  Can you look for better references for Arista and Cisco?  I could not find information regarding 'consumed capacity' of IC to FABRIC in the references provided.

Reply:
In Nokia SRLinux we provide this utilization data per direction (ingress/egress): https://documentation.nokia.com/srlinux/23-10/books/data-model-reference/srl_nokia-platform_0.html#d6179